### DISCUSSION TITLE
Pls provide me the access

### DISCUSSION THREAD

**Discussion by AgnelJ** [appium/appium#20567] [General]
Pls provide me the access

**Comment by mykola-mokhnach** [comment]
Do you mean the access to Appium forum? What is your user name there?

**Reply by AgnelJ** [reply]
Yeah

**Reply by khemalathaa** [reply]
> Do you mean the access to Appium forum? What is your user name there?

I need access for Appium forum as well. I have signed up but I didn't receive any email for activation. I checked it in spam as well. I tried with my another email as well. But no use, Could you please provide me access for appium forum? I entered my name as Kri while Sign up.
